samples/main.cpp

```diff
@@ -519,7 +533,11 @@ int main(int, char**)
 		return -1;
 	}
 
-	glfwGetWindowContentScale(g_mainWindow, &s_displayScale, &s_displayScale);
+#ifdef __APPLE__
```

Comment:
This is disappointing since GLFW is cross platform.

Reply:
Yes it is. I tried to handle DPI without ifdefs, but so far I didn't found the way.